_tuning-your-cluster/availability-and-recovery/workload-management/wlm-feature-overview.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: Workload management
+nav_order: 70
+has_children: true
+parent: Availability and recovery
+---
+
+# Workload management
+
+Workload management allows users to group and search network traffic, isolating system resources to prevent the overuse of network resources by specific requests. It offers the following benefits:
```

Comment:
```suggestion
Workload management allows users to group search traffic and isolate system resources, preventing the overuse of network resources by specific requests. It offers the following benefits:
```

Reply:
@kaushalmahi12: How about this? The other two sentences from the original request are in the bulleted list below.

Reply:
it seems fine, we just need to replace the `network` with `system` . since the network is just one part of system resources.